go/client/cmd_config.go

```diff
@@ -127,27 +136,61 @@ func (v *CmdConfigInfo) ParseArgv(ctx *cli.Context) error {
 }
 
 func (v *CmdConfigGet) Run() error {
-	cli, err := GetConfigClient(v.G())
-	if err != nil {
-		return err
-	}
-	var val keybase1.ConfigValue
-	val, err = cli.GetValue(context.TODO(), v.Path)
-	if err != nil {
-		return err
-	}
 	dui := v.G().UI.GetDumbOutputUI()
-	switch {
-	case val.IsNull:
-		dui.Printf("null\n")
-	case val.I != nil:
-		dui.Printf("%d\n", *val.I)
-	case val.S != nil:
-		dui.Printf("%q\n", *val.S)
-	case val.B != nil:
-		dui.Printf("%t\n", *val.B)
-	case val.O != nil:
-		dui.Printf("%s\n", *val.O)
+	if v.Direct {
```

Comment:
can you split this up into 2 different functions?

Reply:
Done, thanks!